services/galley/src/Galley/API/Teams/Features.hs

```diff
@@ -899,6 +903,13 @@ instance SetFeatureConfig db ExposeInvitationURLsToTeamAdminConfig where
       LockStatusLocked -> throwS @OperationDenied
       LockStatusUnlocked -> persistAndPushEvent @db tid wsnl
 
+instance SetFeatureConfig db OutlookCalIntegrationConfig where
+  setConfigForTeam tid wsnl = persistAndPushEvent @db tid wsnl
```

Comment:
I think, we don't want to check if the feature is locked here. This is an internal function and the necessary checks should already been made before this is called. And specifically for `PATCH` we want to allow overriding the team feature because this is used by IBIS.

Reply:
then why does the instance above do it?  and why would ibis be allowed to break locks?  ah, i see.  it's locked for team admins, but ibis still needs to touch it.

oh dear what a mess.

Reply:
I don't understand why the function above does the check, either. I am currently trying to clarify this. Maybe it is a mistake.

It used to be much messier, believe me :)